command/ca/renew.go

```diff
@@ -514,17 +514,17 @@ func (r *renewer) RenewAndPrepareNext(outFile string, expiresIn, renewPeriod tim
 
 	// Get next renew duration
 	next := nextRenewDuration(resp.ServerPEM.Certificate, expiresIn, renewPeriod)
-	Info.Printf("%s certificate rekeyed, next in %s", resp.ServerPEM.Certificate.Subject.CommonName, next.Round(time.Second))
+	Info.Printf("%s certificate renewed, next in %s", resp.ServerPEM.Certificate.Subject.CommonName, next.Round(time.Second))
```

Comment:
shouldn't this be `rekeyed`?

Reply:
Getting the daemon mode to work we made a new function for rekey since we had to pass in different parameters instead of bloating up `RenewAndPrepareNext()` we separated it into a new `RekeyAndPrepareNext()`  So there is still the renew functions there that print out has been renewed and the new ones for rekey that print out rekey.